### PR TITLE
1.0.0 beta.4 proposed update

### PR DESCRIPTION
**Pull request by lifflander**
Beta.4 with all features!

[1.0.0-beta.4 Status](http://www.red.af/analyzeUrl?tag=origin/1.0.0-beta.4-proposed-update&label=1.0.0-beta.3&label=1.0.0-beta.2&label=1.0.0-beta.1&label=1.0.0-alpha&label=1.0.0-beta.4)

### CODE REVIEW

**Comment by nlslatt** [comment]
Builds standalone on Mutrino and passes all tests there.  I haven't done any testing within EMPIRE yet (use EMPIRE branch 1116-update-vt-to-1-0-0-beta-4).

**Comment by nlslatt** [comment]
On Mutrino, the only EMPIRE test failures with vt are either ones we don't support or ones that are also failing under MPI.

**Comment by nlslatt** [comment]
> On Mutrino, the only EMPIRE test failures with vt are either ones we don't support or ones that are also failing under MPI.

Except for problems related to the LB config file.

**Comment by lifflander** [comment]
The problems with the LB file have now been fixed on this branch along with #561

File: src/vt/configs/arguments/args.cc [hunk 1]
```diff
@@ -54,9 +54,11 @@ namespace vt { namespace arguments {
 
 /*static*/ bool        ArgConfig::vt_color              = true;
 /*static*/ bool        ArgConfig::vt_no_color           = false;
-/*static*/ bool        ArgConfig::vt_auto_color         = false;
 /*static*/ bool        ArgConfig::vt_quiet              = false;
 
+/*static*/ int32_t     ArgConfig::vt_sched_num_progress = 2;
+/*static*/ bool        ArgConfig::colorize_output       = false;
```
Comment (pnstickne) [inline src/vt/configs/arguments/args.cc]: Got moved down a bit D:

Comment (lifflander) [reply src/vt/configs/arguments/args.cc]: I created an issue for this to resolve in beta.5: https://github.com/DARMA-tasking/vt/issues/586

**Pull request merged**